libnmstate/varlink/nmstate_varlink.py

```diff
@@ -0,0 +1,241 @@
+import os
```

Comment:
I renamed directories as libnmstate/nmstate_varlink and tests/nmstate_varlink because naming it as varlink causes module conflicts with dependency varlink module.

Reply:
How exactly does it conflict? Please provide a specific example.

Reply:
When importing "varlink" module from the implemented files(libnmstate/varlink and tests/varlink) it does relative import from the mentioned files because it's named "varlink" instead of the varlink dependency. Using "absolute_import" didn't work. Is there any other ways?

Reply:
Can you please publish the problematic code in a branch, then I will try  to take a look in the next days.

Reply:
@tyll 
I tried to recreate the problem and only naming tests/varlink raises the problem. If I name it has tests/varlink_test it works. Is it okay ?
 
Please below look below comment also. I need for suggestion for that issue.

Reply:
`tests/integration/varlink_service_test.py` seems to be a good place and should not conflict.